app/routes/user.js

```diff
@@ -15,6 +15,7 @@ export default Route.extend({
         return this.store.queryRecord('user', { user_id }).then(
             user => {
                 params.user_id = user.get('id');
+                params.include_yanked = 'n';
```

Comment:
Should we just make this the default and not specify it in the URL here?

Reply:
Here was my thinking. If it's not the direction we want to go, I can make changes.

There are situations where we ***want*** to show yanked crates: https://github.com/rust-lang/crates.io/issues/958#issuecomment-351421886, https://github.com/rust-lang/crates.io/issues/145#issuecomment-281492212.

Given the following two URLs:

* `/api/v1/crates?search=foo`
* `/api/v1/crates?user_id=2576`

One option would be to add conditional logic to the controller like "if `user_id` is set, then don't show yanked crates; if `search` is set, then show yanked crates".

I shied away from that option because I think it would be confusing. A contributor focused on the front end would have to keep in their head which rules apply to which parameters. 

Given the following two URLs:

* `/api/v1/crates?search=foo&include_yanked=yes`
* `/api/v1/crates?user_id=2576&include_yanked=no`

It's much less surprising what will be returned from these endpoints.

However, I'm not sure we want to audit and edit every location where the front end calls the `/api/v1/crates` endpoint – and we can't audit third-party crawlers. Beacuse `/api/v1/crates` has historically defaulted to showing yanked crates, I opted to only add `&include_yanked=no` to the specific front end code where we don't want to show yanked crates.

* `/api/v1/crates?search=foo`
* `/api/v1/crates?user_id=2576&include_yanked=no`

This makes the URLs for the API endpoint a little bit more ugly for these pages. However, this largely only shows up in developer tools. This change doesn't add a query parameter to the URL for a user/team page, e.g. a user page continues to be `/users/sgrif`, not ~`/users/sgrif?include_yanked=no`~.

Am I making any sense? Hopefully I didn't answer a question you weren't asking!